app/templates/frameworks/dashboard.html

```diff
@@ -56,9 +56,46 @@ <h2 class="summary-item-heading">{{ framework.name }} is closed for applications
         </div>
       </div>
     </div>
-  <nav role="navigation" class="framework-application-reference">
+  {% elif framework.status == 'standstill' %}
+    <div class="summary-item-lede">
+      <div class="grid-row">
+        <div class="column-two-thirds">
+          {% if application_made %}
+            <p>You made your supplier declaration and submitted {{ counts.complete }} {{ 'service' if counts.complete == 1 else 'services' }}.</p>
+          {% else %}
+            <p>You didn't submit an application.</p>
+          {% endif %}
+        </div>
+      </div>
+    </div>
   {% endif %}
     <ul class="browse-list">
+
+      {% if framework.status == 'standstill' and application_made %}
+        <li class="browse-list-item framework-application-section">
+          <div class="grid-row">
+            <div class="column-two-thirds">
+              <a class="browse-list-item-link" href="{{ url_for('.download_agreement_file', framework_slug=framework.slug, document_name='result-letter.pdf') }}">
+                <span>Download your application result letter</span>
+              </a>
+              <p>This letter informs you if your {{ framework.name }} has been successful.</p>
```

Comment:
Missing the word `application` after `{{ framework.name }}` in this sentence I think.